doc/programming/parsing.rst

```diff
@@ -1037,6 +1037,25 @@ range to an enum, using ``$$`` to access the parsed value:
     :exec: printf '\x21' | spicy-driver %INPUT
     :show-with: foo.spicy
 
+When parsing a bitfield, you can enforce expected values for some
```

Comment:
Let's use a ref to the bitfield types section around here so users can have a look at more details.